### DISCUSSION TITLE
Bridging with "Export Posts" Plugin

### DISCUSSION THREAD

**Discussion by effone** [mybbgroup/Bookmark-Post#8] [Ideas]
What about adding a feature in Export Posts plugin.
The plugin will check whether "Bookmark Post" plugin is installed and allow a user to downdoad the posts he has bookmarked...

**Comment by lairdshaw** [comment]
I like this idea.

UI possibilities:
1. On the "Export Posts" page - usercp.php?action=exportposts - add a toggle "Export my posts" versus "Export bookmarked posts".
2. In the UCP sidebar under "Export Posts", change the existing sub-item "Export Posts" to "Export My Posts", and add a sub-item "Export Bookmarked Posts". Clicking on either sub-item will load the "Export Posts" page with the associated toggle checked.

Yes?

**Reply by effone** [reply]
I am yet to dig down the plugin but Yes, that sounds like a neat idea.
